### ISSUE TITLE
Consistent terminology for topic configs

### ISSUE THREAD

**Issue by tombentley** [eyefloaters/console#142]
* When creating a topic we have an "Options" page which talks about "topic configuration options"
* On the topic page there is a tab for "Configuration" (which I assume will eventually show the topic config options)
* In the hamburger menu on the topics page we have "Edit properties"

These are all referring to the same conceptual thing – topic config parameters/options – so we should use consistent terminioogy

**Comment by riccardo-forina** [comment]
Agree. Do we have a preference?

**Comment by tombentley** [comment]
"Topic configs" would be the usual term in the AK community. But it's slightly awkward because it's an abbreviation and it's being used to refer to the many options of a single topic, where as "Topic configurations" in the normal English meaning would mean the configuration of many topics. Personally I've sometimes used "configuration parameters".

My suggestion would be:

* "configuration parameters* in the wizard
* "configuration" for the tab in the topic panel
* "edit configuration" in hamburger

**Comment by shirimordechay** [comment]
I agree, Configuration fits best here and that is how it's called on the Kafka topic operator in OCP as well. 
I think "Configuration" in the wizard and topic tab and "edit configuration" from the topic kebab menu.
